styles/japanesewithanime.com/catppuccin.user.css

```diff
@@ -0,0 +1,397 @@
+/* ==UserStyle==
+@name japanesewithanime.com Catppuccin
+@namespace github.com/catppuccin/userstyles/styles/japanesewithanime.com
+@homepageURL https://github.com/catppuccin/userstyles/tree/main/styles/japanesewithanime.com
+@version 0.0.1
+@updateURL https://github.com/catppuccin/userstyles/raw/main/styles/japanesewithanime.com/catppuccin.user.css
+@supportURL https://github.com/catppuccin/userstyles/issues?q=is%3Aopen+is%3Aissue+label%3Ajapanesewithanime.com
+@description Soothing pastel theme for japanesewithanime.com
+@author Catppuccin
+@license MIT
+
+@preprocessor less
+@var select lightFlavor "Light Flavor" ["latte:Latte*", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha"]
+@var select darkFlavor "Dark Flavor" ["latte:Latte", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha*"]
+@var select accentColor "Accent" ["rosewater:Rosewater", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve", "red:Red*", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue", "sapphire:Sapphire", "sky:Sky", "lavender:Lavender", "subtext0:Gray"]
+@var select linkColor "Link color" ["rosewater:Rosewater", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve", "red:Red", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue*", "sapphire:Sapphire", "sky:Sky", "lavender:Lavender", "subtext0:Gray"]
```

Comment:
We should use either accent color or blue and lavender as per the style guide.

Reply:
i don't see why the color of links couldn't be an option since it defaults to blue.
the reason i added it as in option is that the titles are links, and the writing has a lot of links.
is there a specific reason why the user shouldn't be able to change link color?

the titles:
![titles](https://raw.githubusercontent.com/BPplays/images/main/pr/jwa/linkcol/titl.webp)

the writing:
![writing](https://raw.githubusercontent.com/BPplays/images/main/pr/jwa/linkcol/writn.webp)

also where in the style guide does it say accent color can be used as link color?

Reply:
It doesn't it's more something we do in userstyles. Not sure how it started but that's usually how we do things. (It's probably a mix of some websites using the "primary color" CSS variable for links, people not reading the style guide from before this repository started, websites having few places to use the accent color, etc.) We've had some discussion recently but currently no changes are planned.

Reply:
I'd like accent colour for links to be the exception instead of the norm. I believe @isabelroses's reasoning is that accent links made a bit more sense to implement for something like a search engine port.

I'd prefer if we kept links to blue aside from exceptions like the above 🤔

Reply:
I'd estimate the majority of userstyles at the moment use accent for links. If this is something you want to stick by it might be worth creating a tracking issue for locating and updating this in all applicable userstyles.

On this PR specifically, let's use the blue links / style guide spec here and remove the link color option.

Reply:
I'm happy to have my mind changed by yourself and @isabelroses if you guys feel particularly strong about it, can discuss further on discord and summarise in a tracking issue if it comes to that.